File: src/app/invoice.ts
export interface Invoice {
   mobileNumber: string;
  invoiceNo: number;
  customerID: number;
  itemDescription: string;
  price: number;
  quantity: number;
  totalAmount: number;
  gst: number;

  flashVideoURL?: string;
  headerImageList?: string[];
  billPDFFile?: string;
  facebookURL?: string;
  instaURL?: string;
  twitterURL?: string;
  footerImageList?: string[];
}

File: Models/Invoice.cs
using System.ComponentModel.DataAnnotations;

namespace UserRetailEBillDisplay.Models
{
    public class Invoice
    {
        [Key]
        public string? MobileNumber { get; set; }
        public int InvoiceNo { get; set; }
        public int CustomerID { get; set; }
        public string? ItemDescription { get; set; }
        public decimal Price { get; set; }
        public int Quantity { get; set; }
        public decimal TotalAmount { get; set; }
        public decimal GST { get; set; }
    }
}
